### DISCUSSION TITLE
4.2.3 won't install

### DISCUSSION THREAD

**Discussion by simon20993** [OnionUI/Onion#1290] [Support]
Hi,

I've tried a few different ways. OTA updates are just pulling 4.2.0 as the latest version and failing to download/install over Wifi. And having tried 3 different SD cards all re-formatted and the 4.2.3 files extracted to the root, all I get when booting up the device is the menus and no install screen etc.

Is this a device issue? 

This is for the Miyoo Mini Plus.

I'm lost for ideas on what to try or do. I've even done a factory reset with this but all I can get is the default Miyoo dashboard and not Onion OS. For reference, the previous beta version worked perfectly fine!

Thanks

**Comment by thomasklemenc** [comment]
Hi,

did you get any other version of Onion to work?
Did you double check that you format the cards to FAT32?

If all you ever get is the default Miyoo menu, even with different SD cards chances are there is a device issue. maybe a bad solder joint on the SD slot or similar
